cmd/swarmctl/cluster/update.go

```diff
@@ -114,6 +119,11 @@ var (
 				if err != nil {
 					return err
 				}
+
+				if hbPeriod < time.Duration(0) {
```

Comment:
This should be checked somewhere here:

https://github.com/docker/swarmkit/blob/master/manager/controlapi/cluster.go#L40